docs/context/spec.md

```diff
@@ -80,6 +80,13 @@ The identifier "foo" is proprietary, an application that can use it is free to d
 The following are standard FDC3 context types.
  __Note:__ The specification for these types are shared with the [FINOS Financial Objects](https://fo.finos.org) definitions, JSON schemas are hosted with FDC3.
```

Comment:
Note for @greyseer256: this sentence is no longer correct once we merge in new types - and that group has changed focus/no longer maintains those definitions. Hence, its probably time to drop this